### PR TITLE
Allow widgets and charts to re-render with query updates

### PR DESCRIPTION
**Pull request by mateoclarke**
Fixes https://github.com/cityofaustin/atd-data-tech/issues/3607

Sep-24-2020 17-10-46

### CODE REVIEW

**Review by mddilley — APPROVED** [review]
Thanks for sharing about this one yesterday. It was great to reflect on the initial code that we wrote with hooks, and I'm glad to see that you improved the original code and comments. If we pull this out as a Kickstand component, I think that it will be a good chance to reassess some of the logic even more.

I tested it locally, and everything looks great. 🙌

File: atd-vze/src/Components/GridTable.js [hunk 1]
```diff
@@ -142,7 +143,7 @@ const GridTable = ({
         setChartQuery(updatedChartsQuery);
       }
     }
-  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+  }, [query.gql]); // eslint-disable-line react-hooks/exhaustive-deps
```
Comment (sergiogcx) [inline atd-vze/src/Components/GridTable.js:146]: We should probably remove the linter rule exception

**Review by sergiogcx — APPROVED** [review]
Fine with me, but I think that eslint exception needs to be removed if no longer needed

File: atd-vze/src/Components/GridTable.js (hunk 1 above)
Comment (mateoclarke) [reply atd-vze/src/Components/GridTable.js:146]: it is needed. It expects a longer list of dependencies thats unnecessary imho.

**Pull request merged**